src/moon.c

```diff
@@ -0,0 +1,225 @@
+#include <predict/predict.h>
+#include "unsorted.h"
+#include <stdlib.h>
+#include <string.h>
+#include "defs.h"
+#include "sun.h"
+
+/**
+ * This function reduces angles greater than two pi by subtracting two pi
+ * from the angle
+ *
+ * \copyright GPLv2+
+ **/
+double FixAngle(double x)
+{
+
+	while ( x > 2*M_PI )
+		x-=2*M_PI;
+
+	return x;
+}
+
+/**
+ * This function is used in the FindMoon() function.
+ *
+ * \copyright GPLv2+
+ **/
+double PrimeAngle(double x)
+{
+	x=x-360.0*floor(x/360.0);
+	return x;
+}
+
+/* This function determines the position of the moon, including the azimuth and elevation headings, relative to the latitude
```

Comment:
This is a very long line. Should we try to limit ourselves to 80 chars? That's the default in most code formatting tools, narrow enough to be readable, and mostly wide enough to not cause crazy line breaking madness.